cairo-toolchain/SKILL.md

```
@@ -0,0 +1,286 @@
+asdf global starknet-foundry 0.56.0
+
+# Or install directly
+curl -L https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scripts/install.sh | sh
+snfoundryup
+```
+
+### .tool-versions
+
+Pin versions for reproducible builds:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)